asset/sass/blocks/_navbar.scss

```diff
@@ -0,0 +1,4 @@
+.navbar {
+    border-top : 3px solid darken($brand-primary, 10%);
+    /*box-shadow : 0 0 9px 0 rgba(0, 0, 0, .3);*/
+}
```

Comment:
as above: empty line at the end of the file, please

... and other files, not going to point it anywhere else